Dockerfile.prod

```diff
@@ -53,6 +53,9 @@ RUN cd apps/fz_http/assets \
 # Compile the release
 RUN mix compile
 
+# Gen SAML certs
```

Comment:
We probably want to do this using one of the `bin/` handlers like we do the other secrets right?

Reply:
i.e. these would live outside the docker image and be different for each user

Reply:
Right, for prod, yes. I forgot about officially distributed images, and thought the image was always rebuilt.

Reply:
The final image won't have access to mix tasks though. Need to write some extra code.

Reply:
Ah yeah, that's right. These are just standard PEM-encoded certs right? Probably just makes sense to just document using `openssl` to generate them instead of `mix phx.gen.cert` or writing our own code to do it. I would say use the openssl in the docker image but it'll be minimized and won't have it (right?)

Reply:
The prod image actually does have openssl. So we could provide a generate-saml-cert bin script.